packages/material-design-lite/material-design-lite.scss

```diff
@@ -15,6 +15,7 @@
  */
 
 @import "mdl-animation/mdl-animation";
+@import "mdl-button/index";
```

Comment:
oh whoops. Yea, I'm thinking of the JS-side where we use `index` for entry points.